### DISCUSSION TITLE
"No extensions found" when trying to install

### DISCUSSION THREAD

**Discussion by Pablo1107** [community/community#6622] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/17091659/138521334-f818893a-7671-491c-a08f-22b1da471b2d.png)
![image](https://user-images.githubusercontent.com/17091659/138521370-7a449e25-6433-426b-829b-0cf8d559f4f8.png)

As indicated in the Marketplace site, I did Ctrl-P and paste `ext install GitHub.copilot` which makes a marketplace search which found nothing.

**Comment by sheepy0125** [comment]
Are you using VSCodium? You could try downloading the VSIX extension extension (from [this page](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot), `Download Extension` on the right side) and hitting `Ctrl+Shift+P` and `Extensions: Install from VSIX`.

**Comment by Pablo1107** [comment]
I installed it from the Arch repositories using `sudo pacman -S code` (actually I'm a vim user, but I want to try this out). I'll try install it as you say.

**Comment by vdoubleu** [comment]
I have the same issue and tried the solution proposed by @sheepy0125. However, after installing, I get another error saying that copilot requires vscode version 1.57+. After running `code -v` i see that I have version 1.61. I tried reinstalling vscode, and also updating the entire system but still no luck. I still have that same error. I also installed vscode through `sudo pacman -S code`

**Reply by vdoubleu** [reply]
ah nevermind, realised this question was already answered here: https://github.com/github/feedback/discussions/6629
